docker/conf/docker-compose.demo.yml.dist

```diff
@@ -0,0 +1,61 @@
+version: "3.4"
+services:
+    postgres:
+        image: postgres:10.5-alpine
+        volumes:
+            - ./docker/postgres/postgres.conf:/var/lib/postgresql/data/postgresql.conf
+        environment:
+            - PGDATA=/var/lib/postgresql/data/pgdata
+            - POSTGRES_USER=root
+            - POSTGRES_PASSWORD=root
+            - POSTGRES_DB=shopsys
+
+    webserver:
+        image: nginx:1.13-alpine
+        depends_on:
+            - php-fpm
+        volumes:
+            - web-volume:/var/www/html/web
+            - ./docker/nginx/nginx.conf:/etc/nginx/conf.d/default.conf
```

Comment:
configurations are mounted from workspace outside of the image
i think you can mount the same way web-volume so we do not need to do any of the thing i mentioned above

Reply:
@MattCzerner So should I mark all config mounts as read-only?

Reply:
Done